ethers-contract/ethers-contract-abigen/src/contract/events.rs

```diff
@@ -378,6 +393,49 @@ mod tests {
         Context::from_abigen(Abigen::new("TestToken", "[]").unwrap()).unwrap()
     }
 
+    fn test_context_with_alias(sig: &str, alias: &str) -> Context {
+        Context::from_abigen(
+            Abigen::new("TestToken", "[]")
+                .unwrap()
+                .add_event_alias(sig, alias),
+        )
+        .unwrap()
+    }
+
+    #[test]
```

Comment:
This will prevent rustfmt from messing up the quote
```suggestion
    #[test]
    #[rustfmt::skip]
```